index.html

```diff
@@ -5,9 +5,50 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Acme&family=Fredoka:wght@300;400;500&family=Mukta:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Plantcare</title>
   </head>
   <body>
-    <p class="test">OLa</p>
+    <header class="header">
+      <p class="logo">Plant<span>Care</span></p>
+      <nav class="header__navigation">
+        <ul class="header__navigation--list">
+          <li class="header__navigation--item">
+            <a href="#" class="header__navigation--link">Text 1</a>
+          </li>
+          <li class="header__navigation--item">
+            <a href="#" class="header__navigation--link">Text 2</a>
+          </li>
+          <li class="header__navigation--item">
+            <a href="#" class="header__navigation--link">Text 3</a>
+          </li>
+        </ul>
+        <button class="btn btn__orange">
+          <a href="#" class="btn-link"> Log in </a>
+        </button>
+      </nav>
+    </header>
+
+    <main>
+      <section class="hero grid-16-cols">
+        <div class="hero__content">
+          <h1 class="heading-primary">Know what you plants really need</h1>
```

Comment:
I see that the `really` text is not in orange color as per designed approved.  Could you  fix it.